torch/_torch_docs.py

```diff
@@ -4888,47 +4882,44 @@ def merge_dicts(*dicts):
           double backward will usually do an additional backward through `U` and
           `V` even if the original backward is only on `S`.
 
-.. note:: When :attr:`some` = ``False``, the gradients on ``U[:, min(n, m):]``
-          and ``V[:, min(n, m):]`` will be ignored in backward as those vectors
+.. note:: When :attr:`some` = ``False``, the gradients on :code:`U[..., :, min(m, n):]`
+          and :code:`V[..., :, min(m, n):]` will be ignored in backward as those vectors
           can be arbitrary bases of the subspaces.
 
-.. note:: When :attr:`compute_uv` = ``False``, backward cannot be performed since ``U`` and ``V``
+.. note:: When :attr:`compute_uv` = ``False``, backward cannot be performed since `U` and `V`
           from the forward pass is required for the backward operation.
 
 Args:
-    input (Tensor): the input 2-D tensor
+    input (Tensor): the input tensor of size :math:`(*, m, n)` where `*` is zero or more
+                    batch dimensions consisting of :math:`m \times n` matrices.
     some (bool, optional): controls the shape of returned `U` and `V`
+    compute_uv (bool, optional): option whether to compute `U` and `V` or not
     out (tuple, optional): the output tuple of tensors
 
 Example::
 
-    >>> a = torch.tensor([[8.79,  6.11, -9.15,  9.57, -3.49,  9.84],
-                          [9.93,  6.91, -7.93,  1.64,  4.02,  0.15],
-                          [9.83,  5.04,  4.86,  8.83,  9.80, -8.99],
-                          [5.45, -0.27,  4.85,  0.74, 10.00, -6.02],
-                          [3.16,  7.98,  3.01,  5.80,  4.27, -5.31]]).t()
-
+    >>> a = torch.randn(5, 3)
```

Comment:
I think it's helpful having an actual tensor in the doc that can be copy/pasted to reproduce the same documented output

Reply:
Sure, I will make that chane.